### ISSUE TITLE
Telegram share link url

### ISSUE THREAD

**Issue by vaniwiki** [ComboStrap/combo#101]
https://github.com/ComboStrap/combo/blob/ec02c606c31696f50f735fc3d9cf5e07866cbb10/resources/dictionary/brands.json#L150

Works if the url is set to `https://t.me/`, not `https://telegram.me`. Would clone and open pull request, but this seemed easier...

**Comment by gerardnico** [comment]
It works for me on the iOS mobile. I have just tested.

They have it with the t.me format on the doc.

https://core.telegram.org/widgets/share

If you want to change it for fashion reason, don't hesitate to make a pr.

Thanks !

**Comment by gerardnico** [comment]
This is inspired by

https://ellisonleao.github.io/sharer.js/#telegram

And 

https://sharingbuttons.io/

I got the url from there I think. t.me is knew.

**Comment by gerardnico** [comment]
It does a redirect to telegram.com, no?
I can't see, no computer for now.

**Comment by vaniwiki** [comment]
When I first tried to share with Telegram, I got the message that "this
user doesn't exist. After changing, it worked for me. I'm on iPhone and Mac
.That's why I mentioned, for fashion statement I would not bother.

**Comment by vaniwiki** [comment]
Sorry, my bad. Still doesn't work on iPhone (latest OS). Probably worked on Mac before then, because that works now at least. Not sure how to classify this bug then. More info needed?

**Comment by gerardnico** [comment]
It’s not a bug. It’s just that combostrap does not have a social telegram
account. No?

**Comment by vaniwiki** [comment]
No, this is for the share icons, your social plugin. Not your website. I want to share on Telegram, Facebook, Twitter, etc. All of them worked fine, except Telegram which on iPhone told "No such User Exists." It doesn't even say which user doesn't exist. Since I'm an avid fan of Telegram, that is the one that I would use, mainly. Strange it works on one OS and not the other...

**Comment by gerardnico** [comment]
I use

https://ellisonleao.github.io/sharer.js/#telegram

If you have the same behaviour there, there is nothing I can do.

**Comment by gerardnico** [comment]
They use the short version ;)

https://github.com/ellisonleao/sharer.js/blob/main/sharer.js#L115

**Comment by gerardnico** [comment]
Did you try manually?
May be it's just telegram?

**Comment by vaniwiki** [comment]
Yes, I think this is one issue for Telegram, in that case.
